rosa_tutorials/rosa-mobb-azure-groupsync.adoc

```diff
@@ -0,0 +1,176 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-azure-groupsync"]
+= Tutorial: Using Group Sync Operator with Azure Active Directory and ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-azure-groupsync
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-18
+// ---
+// date: '2021-11-08'
+// title: Using Group Sync Operator with Azure Active Directory and ROSA
+// tags: ["Azure", "ROSA"]
+// authors:
+//   - Steve Mirman
+//   - Kumudu Herath
+// ---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+This guide focuses on how to synchronize Identity Provider (IDP) groups and users after configuring authentication in OpenShift Cluster Manager (OCM). For an IDP configuration example, please reference the link:https://mobb.ninja/docs/idp/azuread[Configure Azure AD as an OIDC identity provider for ROSA/OSD] guide.
+
+// ALLOW PR#64888 TO MERGE AND REPLACE THIS URL WITH: xref:../rosa_tutorials/rosa-mobb-azure-oidc[Configure Azure AD as an OIDC identity provider for ROSA/OSD] guide.
+
+To set up group synchronization from Azure Active Directory (AD) to ROSA/OSD, you must:
```

Comment:
Should we refer to ROSA only (since that's where we're publishing this), or is "ROSA/OSD" ok?